File: src/main/java/com/claudio_personagens/bookingcharactersapi/domain/Address.java
package com.claudio_personagens.bookingcharactersapi.domain;

import com.claudio_personagens.bookingcharactersapi.base.AbstractBaseEntity;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Column;
import javax.persistence.Entity;

@Data
@Entity
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Address extends AbstractBaseEntity {
    @Column(length = 150)
    private String street;
    @Column(length = 5)
    private String number;
    @Column(length = 100)
    private String district;
    @Column(length = 100)
    private String complement;
    @Column(length = 100)
    private String city;
    @Column(length = 100)
    private String state;
}

File: src/main/java/com/claudio_personagens/bookingcharactersapi/domain/Address.kt
package com.claudio_personagens.bookingcharactersapi.domain

import com.claudio_personagens.bookingcharactersapi.base.AbstractBaseEntity
import javax.persistence.Column
import javax.persistence.Entity

@Entity
data class Address (
    @Column(length = 150)
    val street: String?,

    @Column(length = 5)
    val number: String?,

    @Column(length = 100)
    val district: String?,

    @Column(length = 100)
    val complement: String?,

    @Column(length = 100)
    val city: String?,

    @Column(length = 100)
    val state: String?
) : AbstractBaseEntity()